### ISSUE TITLE
Create issues even when workflow is invalid

### ISSUE THREAD

**Issue by ianlewis** [slsa-framework/example-package#29]
If the path or names of inputs to the reusable workflow are not correct then the e2e test workflow doesn't run at all and  doesn't create GitHub issues. We need to find a way to have issues created even when the workflow is invalid.

**Comment by laurentsimon** [comment]
definitely helpful. I did not do it to simplify the workflows, but I agree it's needed to catch other errors.

**Comment by ianlewis** [comment]
Yeah, right now if you do a reusable workflow call and the inputs dont match up, Github Actions says the whole workflow is invalid and doesn't do anything else and doesn't run our 'if-failed` job so we don't get an issue created. The only notification we get is maybe 1 of tons of spam email from Github Actions telling us the run failed.

**Comment by ianlewis** [comment]
Related to #204 and #202

**Comment by ianlewis** [comment]
`actionlint` should hopefully catch when the workflow itself is invalid though.
